mycli/packages/completion_engine.py

```diff
@@ -80,9 +82,9 @@ def suggest_type(full_text, text_before_cursor):
     return suggest_based_on_last_token(last_token, text_before_cursor, full_text, identifier)
 
 
-def suggest_special(text):
+def suggest_special(text: str) -> list[dict[str, Any]]:
     text = text.lstrip()
-    cmd, _, arg = parse_special_command(text)
+    cmd, _separator, _arg = parse_special_command(text)
```

Comment:
Even if we're not using a var we still need to name it? Was it flagged by mypy?

Reply:
We don't have to here, but I did it for clarity.  There were other cases where `_` was used for two different types in a row, and mypy did get upset.  Instead, mypy should ignore `_`.

Happy to revert if you think that change is a net loss.